api/explore/next_holiday_calc.js

```diff
@@ -0,0 +1,49 @@
+const Hebcal = require('hebcal');
+const moment = require('moment-holiday');
+
+function nextHoliday(tz) {
```

Comment:
what does this function return, a boolean or a next holiday?

Reply:
the next holiday if there is one within the next 20 days, false otherwise.